examples/quickstart/quickstart.py

```diff
@@ -33,7 +33,7 @@ def importer() -> Output(
 
 
 @step
-def trainer(
```

Comment:
Is this a typo or an intended change? :D

Reply:
fixed this